tortoise_interpreter.py

```diff
@@ -36,16 +36,20 @@ def pen_colour(command):
 
 
 def move_north(command):
-    return f'N {command[1]}'
+    return f"Move {command[1]} {determine_if_plural(command)} north."
```

Comment:
This is good stuff, but you can go futher with this idea. Each of these functions (`move_north`, `move_south`, etc) are still very similar. How much could you extract into another function that they each call? What's the only thing that varies between them?